timemachine/cpp/src/kernels/k_nonbonded_pair_list.cuh

```diff
@@ -119,10 +118,10 @@ void __global__ k_nonbonded_pair_list(
     RealType delta_w = w_i - w_j;
     RealType d2ij = delta_x * delta_x + delta_y * delta_y + delta_z * delta_z + delta_w * delta_w;
 
+    RealType u = static_cast<RealType>(0.0);
     // see note: this must be strictly less than
     if (d2ij < cutoff_squared) {
 
-        RealType u;
```

Comment:
I was curious about this change, did something break that require us to do explicit initialization? Before this was uninitialized, because:

1) `compute_electrostatics` runs `u = charge_scale * qij * inv_dij * ebd;` (initialization)
2) `compute_lennard_jones` runs `u += lj_scale * 4 * eps_ij * (sig6_inv_d6ij - 1) * sig6_inv_d6ij;`

Reply:
Because we always set the values in the energy buffer, need to initialize it to zero.